app/helpers/application_helper.rb

```diff
@@ -1,2 +1,14 @@
+# frozen_string_literal: true
+
+# module ApplicationHelper
 module ApplicationHelper
+  # Returns the full title on a per-page basis.
+  def full_title(page_title = '')
+    base_title = 'Ruby on Rails Tutorial Sample App'
```

Comment:
reseach I18n to apply for text that you can see in screen

Reply:
not fixed it yet